src/StudentsController.ts

```diff
@@ -533,20 +533,30 @@ export class StudentsController {
         return text;
     }
 
-    public async sendFile(student: Student, data: { file: string; title: string; filename: string; mimetype: string; tags?: string[] | undefined }): Promise<SchoolFileDTO> {
+    public async sendFile(
+        student: Student,
+        data: { file: string; title: string; filename: string; mimetype: string; tags?: string[] | undefined; subject?: string; body?: string }
```

Comment:
I wonder if the names of the parameters "subject" and "body" and self explanatory. Maybe something like "messageSubject" and "messageBody" would be better? But I will approve anyway. You can decide whether you want to change it.